x-pack/plugins/licensing/common/license_feature.ts

```diff
@@ -4,8 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { License } from './license';
 import { LicenseFeatureSerializer } from './types';
+import { License } from './license';
 
 export class LicenseFeature {
   private serializable: LicenseFeatureSerializer = license => ({
```

Comment:
- license is not used in `serializable`
- does someone use `onObject` ?

Reply:
1. It's meant to suffice the type definition and give clues about how this is used when the implementation is swapped out by the consumer.
2. This was existing functionality in xpack_info, so I believe so.

Reply:
> This was existing functionality in xpack_info, so I believe so.

Could you provide an example where it was declared? I cannot find

Reply:
https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/xpack_main/server/lib/xpack_info.js#L225

https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/xpack_main/server/lib/xpack_info.js#L280

Reply:
sorry, I still can't see where someone redefine serialization with `onObject` 😞